docs/source/serving/openai_compatible_server.md

```diff
@@ -315,6 +315,28 @@ AI21's Jamba-1.5 models are supported.
 Flags: `--tool-call-parser jamba`
 
 
+#### Models with Pythonic Tool Calls (`pythonic`)
+
+A growing number of models output a python list to represent tool calls instead of using JSON. This has the advantage of inherently supporting parallel tool calls and removing ambiguity around the JSON schema required for tool calls. The `pythonic` tool parser can support such models.
+
+As a concrete example, these models may look up the weather in San Francisco and Seattle by generating:
+```python
+[get_weather(city='San Francisco', metric='celsius'), get_weather(city='Seattle', metric='celsius')]
+```
+
+Limitations:
+* The model must not generate both text and tool calls in the same generation. This may not be hard to change for a specific model, but the community currently lacks consensus on which tokens to emit when starting and ending tool calls.  (In particular, the Llama 3.2 models emit no such tokens.)
```

Comment:
I've seen the llama 3.2 models respond with the `<|python_tag|>` token at the beginning which is not handled correctly, see the screenshots earlier in the thread

Reply:
Since option 2 below seems to be the consensus, I'm not planning on handling `<|python_tag|>` right now, but I've added a TODO.